src/ResourceManager/Resources/Commands.Resources/Models.ResourceGroups/ResourceClient.cs

```diff
@@ -313,16 +313,29 @@ public static List<string> ParseDetailErrorMessage(string statusMessage)
         {
             DeploymentExtended deployment;
 
+            // Poll deployment state and deployment operations after RetryAfter.
+            // If no RetryAfter provided: In phase one, poll every 5 seconds. Phase one 
+            // takes 400 seconds. In phase two, poll every 60 seconds. 
```

Comment:
@bganapa out of curiosity, does phase two have a set length? Phase one is 400 seconds, but it doesn't look like phase two has a length and would run indefinitely. Is that what is expected?